### PR TITLE
Add Lisk HD address discovery example

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

**Comment by ethanfrey** [comment]
@Isabello take a look. you'll like it :)

**Comment by webmaster128** [comment]
@Isabello this is how the new Ed25519HdWallet can be used to derive arbitrary SLIP-0010/BIP-0032 paths

File: packages/iov-lisk/README.md [hunk 1]
```diff
@@ -100,6 +100,62 @@ const bytesToPost = Encoding.fromUtf8(liskCodec.bytesToPost(signedTransaction));
...
 ```
 
+## Lisk HD wallets
+
+Lisk codec and Ed25519HdWallet combined allow you to create a pure
+software implementation of the the Lisk wallet on Ledger or Trezor.
+
+### Address discovery
+
+The following code snipped shows hot to implement address discovery.
```
Comment (Isabello) [inline packages/iov-lisk/README.md]: shows how

Comment (webmaster128) [reply packages/iov-lisk/README.md]: thx, done

**Pull request merged**